docs/source/help.rst

```diff
@@ -0,0 +1,58 @@
+Help
+====
+
+Couldn't find what you were looking for?
+The Featuretools community is happy to provide support to users of Featuretools.
+
+
+Discussion
+----------
+
+Conversation happens in the following places:
+
+1.  **Usage questions** are directed to `StackOverflow`_ with the `#featuretools`_ tag.
```

Comment:
no need to have two links to the same place in once sentence. perhaps on keep the stack overflow one and remove it from #featuretools